### DISCUSSION TITLE
Contact Role in NPSP

### DISCUSSION THREAD

**Discussion by AscendSF** [SFDO-Community/Salesforce-Indicators#149] [Ideas]
Is there any way to use this package to identify the Contact's role in the organization? For instance, I have clients that would like to quickly see whether a Contact is a donor, volunteer, or client (tied to a program).

**Comment by tschug** [comment]
Hello!   Yes - this is possible and a great example, as long as there is data on the Contact's record to indicate such a role.

Out of the box, NPSP has several rollup fields for donations which can be used to show if a Contact is a donor.

Volunteering roles are very specific to the organization, such as a board or committee member or someone who participates in volunteer projects (something like V4S Volunteers for Salesforce). This could be a rollup value (including using DLRS), checkbox, or picklist (we will be supporting multi-select / multi-matching values soon for this type of example).

Similarly, this concept would apply to clients (tied to programs from the PMM Program Management Module or Nonprofit Cloud's Program Management) by having a field or a rollup to indicate involvement as a client/recipient.

**Comment by JodieM** [comment]
Good discussion, but can now be closed.
